src/mscorlib/src/System/Globalization/CultureData.cs

```diff
@@ -1000,7 +1000,7 @@ internal static CultureInfo[] GetCultures(CultureTypes types)
 
             if (nativeEnumCultureNames((int)types, JitHelpers.GetObjectHandleOnStack(ref cultureNames)) == 0)
             {
-                return new CultureInfo[0];
+                return Array.Empty<CultureInfo>();
```

Comment:
GetCultures isn't exposed in .NET Core.  And even if it were, it's a relatively rare case to have no cultures returned (it can happen if, for example, you ask for a specific culture type like replacement cultures), and the act of asking for all cultures isn't an operation that typically happens on a hot path; you usually do it only once or only once in a while.  Given all that, and since this array is being returned out of a public API and we don't know how it's being consumed, the risk (even if minimal) isn't worth it and we should revert this change.